src/applications/personalization/common/components/devtools/DevToolsLoader.jsx

```diff
@@ -36,14 +37,11 @@ const DevToolsPanel = ({ devToolsData, show, setShow }) => {
   return (
     <div className={classes} data-testid="devtools-panel">
       <div className="devtools-panel__content">
-        <button
-          type="button"
-          className="devtools-panel__close-button"
+        <VaButton
```

Comment:
I don't think we have general rules for this. 

I kind of like using the VaButton, because it then correlates in a more standard fashion to the 'react way' including using `className` instead of `class` when attaching css classes to the button, which to me seems less error prone.

If you use className on a plain `va-button` it does not work.

Reply:
thanks! i was kinda preferring to always use the react when they exist so we can look up import paths, but i've seen both way used, so wondered if there was a general rule for using one vs other.